assets/scripts/modules/Scroll.js

```diff
@@ -53,6 +29,24 @@ export default class extends module {
         this.scroll?.scrollTo(target, options)
     }
 
+    /**
+    * Observe new scroll elements
+    *
+    * @param $newContainer (HTMLElement)
+    */
+    addScrollElements($newContainer) {
+        this.scroll?.addScrollElements($newContainer)
+    }
+
+    /**
+    * Unobserve scroll elements
+    *
+    * @param $oldContainer (HTMLElement)
+    */
+    removeScrollElements($oldContainer) {
+        this.scroll?.removeScrollElements($oldContainer)
+    }
```

Comment:
Misaligned block comment syntax and invalid parameter tags.

```suggestion
    /**
     * Observe new scroll elements
     *
     * @param {HTMLElement} $newContainer
     */
    addScrollElements($newContainer) {
        this.scroll?.addScrollElements($newContainer)
    }

    /**
     * Unobserve scroll elements
     *
     * @param {HTMLElement} $oldContainer
     */
    removeScrollElements($oldContainer) {
        this.scroll?.removeScrollElements($oldContainer)
    }
```